src/oauth-helpers.js

```diff
@@ -150,6 +127,21 @@ export async function _refreshAuthTokens(refreshToken) {
 
     return items;
   }
+
+  const { access_token, refresh_token, expires_in } = response;
+
+  // Calculate timestamp when OAuth token expires - current time + it's expires_in timestamp
+  const expiresAt = Math.floor(Date.now() / 1000) + expires_in;
+
+  const items = {
+    accessToken: access_token,
+    refreshToken: refresh_token,
+    expiresAt: expiresAt,
+  };
+
+  chrome.storage.sync.set({ authentication: items });
```

Comment:
teeny tiny suggestion: you can rename `items` to `authentication`, and then use: `chrome.storage.sync.set({ authentication })` (or `chrome.storage.sync.set({ authentication: authentication })`. The extra variable name doesn’t bring much to the understanding of the code here

Reply:
👍🏻 🙏🏻